pkg/netpol/eval/resources.go

```diff
@@ -343,3 +343,39 @@ func (pe *PolicyEngine) getDisjointIPBlocks() ([]*k8s.IPBlock, error) {
 	disjointRes := k8s.DisjointIPBlocks(ipbList, []*k8s.IPBlock{newAll})
 	return disjointRes, nil
 }
+
+func (pe *PolicyEngine) getServicePods(svcName, svcNamespace string) ([]*k8s.Pod, error) {
+	svc := pe.getServiceFromServiceNameAndNameSpace(svcName, svcNamespace)
+	// todo: should return error if the service not found?
+	if svc == nil {
+		return nil, fmt.Errorf("service does not exist: %s\\%s", svcNamespace, svcName)
```

Comment:
use `types.NamespacedName` instead of hard-coded str format (check similar usage in other functions)